WordPress/src/main/res/drawable/img_illustration_construct_site_190dp.xml

```diff
@@ -0,0 +1,310 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="190dp"
+    android:height="142dp"
+    android:viewportWidth="190"
+    android:viewportHeight="142">
+    <path
+        android:fillColor="#C5D9ED"
+        android:fillType="evenOdd"
+        android:pathData="M34.739,127.758V31.7h137.585v96.058H69.326z" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M106.474,105.575v2.5h-2.5"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M99.364,108.075H55.566"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M53.26,108.075h-2.5v-2.5"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M50.76,100.308V60.805"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M50.76,58.171v-2.5h2.5"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M57.87,55.671h43.799"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M103.974,55.671h2.5v2.5"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#00000000"
+        android:fillType="evenOdd"
+        android:pathData="M106.474,63.438v39.502"
+        android:strokeWidth="2"
+        android:strokeColor="#FFF"
+        android:strokeLineCap="round"
+        android:strokeLineJoin="round" />
+    <path
+        android:fillColor="#FFF"
+        android:fillType="evenOdd"
+        android:pathData="M103.177,81.402H59.46v-9.947h51.915v1.749a8.198,8.198 0,0 1,-8.198 8.198" />
+    <path
+        android:fillColor="#C3C4C7"
```

Comment:
@SylvesterWilmott - in other vector images we use color references (it would be `gray_10` here for example). Any reason we don't do that here? Examples in [img_illustration_cloud_off_152dp.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/f8ca54caed8114a6399fa8d3ef2807f1ec07ab1e/WordPress/src/main/res/drawable/img_illustration_cloud_off_152dp.xml) or [img_illustration_add_media_150dp.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/f8ca54caed8114a6399fa8d3ef2807f1ec07ab1e/WordPress/src/main/res/drawable/img_illustration_add_media_150dp.xml)

Reply:
Interesting. @theck13 do you know anything about this? Was it done as part of the colour changes?

Reply:
Yes, all vector images should be using `@color/` or `@android:color/` references only.  There are also extra attributes that are unnecessary in this vector file.  All paths should only need the `fillColor` and `pathData` attributes.  Take a look at any of the other `img_illustration_` files for expected format.  It may take some work with the image source file to merge the stroke with the path or separate the stroke into a different path depending on the desired effect.

Reply:
I think I could fix the color references by doing a quick find and replace, but I'm wondering if there's any other way this is usually done?

Reply:
That's the only way I know to do it.

Reply:
Vector file was updated in 866c12a